### PR TITLE
Raj_Assignment_23_Done

### PR DESCRIPTION
**Pull request by RajAJun20**
Assignment_23_P1
Assignment_23_P2

### CODE REVIEW

File: src/rajAJun20/IntersectionOfArrays.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package rajAJun20;
+
+public class IntersectionOfArrays {
+	int countTemp = 0;
+	
+	// Common numbers
+	void getIntersectionOfArray(double[] array1, double[] array2) {
+		double[] tempIntersectionArray = new double[(array1.length)];
```
Comment (AnshuKJun20) [inline src/rajAJun20/IntersectionOfArrays.java:8]: Here you have to first check which array has minimum length, then take  minimum length is length of your tempIntersectionArray[].

**Review by AnshuKJun20 — COMMENTED** [review]
Please work on comments.

File: src/rajAJun20/IntersectionOfArrays.java (hunk 1 above)
Comment (KrishnaKTechnocredits) [reply src/rajAJun20/IntersectionOfArrays.java:8]: @RajAJun20 I am agree with Anshu's comment. Good job @AnshuKJun20

**Review by KrishnaKTechnocredits — APPROVED** [review]
Apart from Array length, Code looks good to me.

**Pull request merged**